tests/Integration/Builder/OpenInvoiceTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Adyen\Tests\Integration\Builder;
+
+use Adyen\Service\Builder\OpenInvoice;
+use Adyen\TestCase;
+
+class OpenInvoiceTest extends TestCase
+{
+    public function testBuildOpenInvoiceLineItem()
+    {
+        $expectedResult = array(
+            'id' => 1,
+            'itemId' => 1,
```

Comment:
id is a string: https://docs.adyen.com/payment-methods/klarna/invoice-lines#payment-request

Reply:
please also change this and the phpdoc: https://github.com/Adyen/adyen-php-api-library/blob/d4d2a27046a8b48c367b6142b4c532b9b71a2d67/src/Adyen/Service/Builder/OpenInvoice.php#L47